.env

```diff
@@ -15,3 +15,6 @@ REACT_APP_FXA_CONFIG=code-manager
 # which violates the CSP.
 INLINE_RUNTIME_CHUNK=false
 
+# CRA forces us to use a variable named `PUBLIC_URL` to set the CDN URL...
+# See: https://facebook.github.io/create-react-app/docs/advanced-configuration
+PUBLIC_URL=https://addons-code-manager.cdn.mozilla.net
```

Comment:
Did you confirm all of these CDN URLs? I'm not sure what they willl be.

Reply:
Yes, it is from: https://github.com/mozilla/addons-code-manager/issues/327#issuecomment-473513809